### PR TITLE
Fixed Ambire Wallet staking and added Ambire TVL for wallet rewards

### PR DESCRIPTION
**Pull request by stojnovsky**
This PR updates Ambire adapter to fix a couple of issues:
- removed AdEx staking the staking calculation, as it's a different product and protocol: counting in the Ambire Wallet staking was wrong in the first place
- added Ambire TVL to adapter. 

##### methodology (what is being counted as tvl, how is tvl being calculated):
The Ambire TVL consists of all eligible tokens locked in Ambire contracts. $WALLET rewards are distributed based on this number.
Eligible tokens: https://help.ambire.com/hc/en-us/articles/4412943620754-What-tokens-are-eligible-considered-for-the-WALLET-rewards-on-Ambire-Wallet-
WALLET token distribution: https://help.ambire.com/hc/en-us/articles/9592967980828-WALLET-token-distribution

Twitter Link:
https://twitter.com/AmbireWallet
List of audit links if any:
https://code4rena.com/reports/2023-05-ambire

Website Link:
https://www.ambire.com/

Coingecko ID (so your TVL can appear on Coingecko):
https://www.coingecko.com/en/coins/ambire-wallet

Coinmarketcap ID (so your TVL can appear on Coinmarketcap): 
https://coinmarketcap.com/currencies/ambire-wallet/

Description/bio (to be shown on DefiLlama):
The first DeFi wallet that combines power, security and ease of use, while also being open-source and non-custodial.

Token address and ticker if any:
Ticker: Wallet
Address: 0x88800092fF476844f74dC2FC427974BBee2794Ae

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/ambire-wallet exports TVL: 
        
 
 
    ethereum-staking          3.23 M
    ethereum                  22.15 M
    staking                   3.23 M
    
    total                    22.15 M

File: projects/ambire-wallet/ambireTVLContracts.js
Comment (g1nt0ki) [inline projects/ambire-wallet/ambireTVLContracts.js:1]: these are multisig addresses generated by ambire? if so, we cant count it as tvl. 

I would kindly request you to remove these from tvl

File: projects/ambire-wallet/helpers.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+          target: ADEX_staking,
+          params: []
+        }
+      ],
+      abi: 'function shareValue() external view returns (uint)',
+      chain: 'ethereum'
+    })
+    const shareValues = {
+      [WALLET_staking]: parseInt(sharedValues.output[0].output) / 1e18,
+      [ADEX_staking]: parseInt(sharedValues.output[1].output) / 1e18
```
Comment (g1nt0ki) [inline projects/ambire-wallet/helpers.js:50]: I too am equally perplexed why we were count adax staking under ambire in the past, is there some link between the two?

**Review by g1nt0ki — CHANGES_REQUESTED** [review]
.

**Comment by g1nt0ki** [comment]
closed as there was no answer

**Pull request closed without merging**